### ISSUE TITLE
How did you transform images (rectify) in the project?

### ISSUE THREAD

**Issue by cagkanciloglu** [johanhenrikssn/optical-music-recognition#1]
Hi @johanhenrikssn,
 
Mine is a simple question actually. How did you transform images for example im10c.jpg to im10s.jpg?
Is it possible for me to do the same thing on matlab?

Thanks

**Comment by petraohlin** [comment]
We didn't do that transformation, it was just a test set of images where the images named with s is scanned (simpler) and the images named c is photographs (more difficult). 

We tried to make the photographs look as similar as possible to the scanned images by cropping out the interesting part. More pre-processing needs to be done that takes care of the different distortions that occurs in the photographs. I will add the report we wrote about the project if you are more interested in the project. 

It definitely is possible for you to do that restoration in matlab, even though we didn't have the time to do it! Good luck!
